docs/Learning-Environment-Design-Agents.md

```diff
@@ -347,7 +344,7 @@ continuous action spaces.
 
 When using Discrete Actions, it is possible to specify that some actions are
 impossible for the next decision. Then the Agent is controlled by an External or
-Internal Brain, the Agent will be unable to perform the specified action. Note
+Learning Brain, the Agent will be unable to perform the specified action. Note
```

Comment:
It should be External that gets corrected to Learning.

Reply:
There is no external brain anymore

Reply:
Yes I know, but you still say 'Then the Agent is controlled by an External or Learning Brain, the Agent will be unable to perform the specified action. Note'